src/plugins/esbuild-alias.js

```diff
@@ -9,12 +9,11 @@ const isFile = path => {
 }
 
 const getAsFile = path => {
-  const hasIndex = /index$/
+  const isDir = fs.existsSync(path) && fs.lstatSync(path).isDirectory()
   const extensions = ['js', 'mjs', 'ts']
```

Comment:
@erikhaandrikman this is slightly different from your solution (PR #110) .. what do you think?

Reply:
We both tried to solve the same issue, i suggest merging this one